src/qibolab/instruments/zhinst/util.py

```diff
@@ -0,0 +1,24 @@
+"""Utility methods."""
+
+from qibolab.qubits import Qubit
+
+SAMPLING_RATE = 2
+NANO_TO_SECONDS = 1e-9
```

Comment:
I would say that whatever conversion we apply, it should happen in a single place. And since that place is not here, then we could even move these definitions (eventually).

I know that right now this is not the case, since there are conversions in all the other modules (namely `pulse`, `sweep`, and `executor`), but given your current experience with Zurich, wouldn't be possible to move them in a single place?

Reply:
`NANO_TO_SECONDS` is used in a lot of places. Most of them have to do with accessing pulse properties. I agree that extracting them into one place will make it easier to decide how to handle conversions in the future, but I am not sure if that's already possible at this stage. I could in principle pre-process all pulses and replace times (start and duration) with seconds, but I think that's bad. I could reuse the `ZhPulse` class to do it, but then I would duplicate parts of pulse definitions, so that's bad as well. I will see if I can find something else.

Reply:
I know that eventually should be done. Whether it is optimal or no to do it now, I'd leave up to you.